owtf/data/conf/framework.yaml

```diff
@@ -0,0 +1,250 @@
+FRAMEWORK_CONFIGS:
```

Comment:
Make this `FRAMEWORK_CONFIG` and convert any old section headers from the cfg file like `# [DICTIONARIES]` to a consistent `# ---------- DICTIONARIES-----------` like comment.